packages/extension-seo-url-mapping/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@salesforce/extension-seo-url-mapping",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "lint": "npm run lint:js",
+    "lint:fix": "npm run lint:js -- --fix",
+    "lint:js": "pwa-kit-dev lint \"**/*.{js,jsx,ts,tsx}\"",
+    "start": "npm --prefix ./dev start",
+    "start:inspect": "npm --prefix ./dev run start:inspect"
+  },
+  "peerDependencies": {
+    "@loadable/component": "^5.15.3",
+    "@salesforce/pwa-kit-extension-sdk": "4.0.0-extensibility-preview.4",
+    "@salesforce/pwa-kit-react-sdk": "4.0.0-extensibility-preview.4",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "peerDependenciesMeta": {
+    "@salesforce/pwa-kit-extension-sdk": {
+      "optional": true
+    },
+    "@salesforce/pwa-kit-react-sdk": {
+      "optional": true
+    }
+  },
```

Comment:
These were removed from the sample extension code (or they should have been) and shouldn't be in your extension. you can remove them.